src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt

```diff
@@ -61,7 +61,11 @@ class EvolvePokemon : Task {
                     if (bot.api.inventories.candyjar.getCandies(pokemonMeta.family) >= pokemonMeta.candyToEvolve) {
                         Log.yellow("Evolving ${it.pokemonId.name} CP ${it.cp} IV ${it.getIvPercentage()}%")
                         val evolveResult = it.evolve()
-                        Thread.sleep(300)
+                        if( settings.evolveTimeDelay > 300){
```

Comment:
no it is not. If someone specifies 0 as the evolve_time_delay, there will be no evolve time delay, which will mess with the stack evolution